components/public-api/gitpod/experimental/v2/authprovider.proto

```diff
@@ -0,0 +1,137 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
```

Comment:
@AlexTugarev after our discussion today I think it is better to add to `gitpod.v1` from the start to avoid breaking the dashboard later

Reply:
ok